NewHaven/test/VGMapDriver.cpp

```diff
@@ -0,0 +1,26 @@
+//
+// Created by Nizz Wizz on 2020-02-05.
+//
+
+#include <iostream>
+#include "../src/VGMap.h"
+#include "boost/graph/graph_utility.hpp"
+
+using namespace std;
+int main() {
+    int playerNum;
+    cin >> playerNum;
+    VGMap* vgMaps = nullptr;
+    vgMaps = new VGMap[playerNum];
```

Comment:
Why not assign the value to the variable directly?

Reply:
Rip indeed it should be lol I'll do all the fixes.